.gitignore

```diff
@@ -25,3 +25,6 @@ cmake-build/
 .vscode/
 .devcontainer/
 .devcontainer.json
+cmake-build/
+build/
```

Comment:
We might want to unify these two at some point, seems kinda weird we could have both in our repo.

Reply:
I kept only cmake-build and out. I don't know where `build` may be used.